lib/trino-filesystem-gcs/src/test/java/io/trino/filesystem/gcs/TestGcsFileSystemGcs.java

```diff
@@ -28,4 +34,16 @@ void setup()
     {
         initialize(getRequiredEnvironmentVariable("GCP_CREDENTIALS_KEY"));
     }
+
+    @Test
+    void testCreateFileRetry()
+    {
+        assertThatNoException().isThrownBy(() -> {
```

Comment:
I'm confused by the purpose of `assertThatNoException()`. Any exception should fail the test. Isn't that the normal behavior of unit tests?

Reply:
I think it captures intent nicely - for the case when you just want to check code runs fine and do not do any assertions. 
Otherwise, if there is code without assertions in test method, it kinda looks that test writer forgot about sth.